documentation/public/user-guide/npm-package/index.md

```diff
@@ -3,5 +3,4 @@
 -   [Installation](./installation.md)
 -   [Using the Parser](./using-the-parser.md)
 -   [Using the Cursor](./using-the-cursor.md)
--   [Using Queries](./using-queries.md)
```

Comment:
i think this line was unintentionally removed?

Reply:
done